src/metabolic_ninja/rabbitmq.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2018, Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions to submit jobs through RabbitMQ."""
+
+import json
+
+import pika
+
+
+def submit_job(**kwargs):
+    """Submit a new job to the message queue."""
+    message = json.dumps(kwargs)
```

Comment:
Might want to force compact JSON here with `separators=(',', ':')` argument. Also, we should make a call on `allow_nan` as it's later disabled in the backend response.

Reply:
Good idea with compact JSON. I don't think NaN is applicable to any of the current job arguments though.

Reply:
What I mean is that `allow_nan=True` [is the default argument](https://docs.python.org/3.6/library/json.html?highlight=json#json.dump) and we explicitly do not allow it later on. So maybe we should also do that within the pipeline?

Reply:
None of the current arguments for job submissions can contain nan values so I don't see a reason to do it explicitly here. Go ahead with it though if you have reason to believe it would prevent any problems.